classes/student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, name, parent_permession, classroom)
+    super(age, name, parent_permession)
```

Comment:
- [ ] Kindly arrange your arguments. The name and parent_permission arguments on the constructor are optional and so they should come last.

- [ ] The parent_permission assignment on line 5 needs to be done correctly.

  ```
  def initialize(classroom, age, name = 'unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
  ```